packages/nitro-protocol/contracts/examples/RockPaperScissors.sol

```diff
@@ -16,69 +17,79 @@ import '../Outcome.sol';
   *
 */
 contract RockPaperScissors is ForceMoveApp {
+    using SafeMath for uint256;
 
-    enum PositionType { Start, RoundProposed, RoundAccepted, Reveal }
-    enum Weapon { Rock, Paper, Scissors }
+    enum PositionType {Start, RoundProposed, RoundAccepted, Reveal}
+    enum Weapon {Rock, Paper, Scissors}
 
-    struct RPSData{
+    struct RPSData {
         PositionType positionType;
         uint256 stake;
         bytes32 preCommit;
-        Weapon bWeapon;
-        Weapon aWeapon;
+        Weapon playerFirstWeapon;
+        Weapon playerSecondWeapon;
         bytes32 salt;
     }
 
     /**
-    * @notice Deocdes the appData.
-    * @dev Deocdes the appData.
-    * @param appDataBytes The abi.encode of a RPSData struct describing the application-specific data.
+    * @notice Decodes the appData.
+    * @dev Decodes the appData.
+    * @param toGameDataBytes The abi.encode of a RPSData struct describing the application-specific data.
```

Comment:
`toGameDataBytes` sounds more like a function name (to me) than a variable. Also, `appDataBytes` is a nitro framework level concept so we should probably use that name here.

I guess you were thinking `to` and `from` for each commitment in the transition. But this function is applied to both. 

I would also like to keep with `app` rather than `game` if possible (throughout).

Reply:
Woops this was a mistake :)
Lots of refactoring